public/leie.html

```diff
@@ -55,12 +55,11 @@
       <h3>Hva slags teknisk utstyr er tilgjengelig?</h3>
       <p>Escape har en god del teknisk utstyr til disposisjon. Blandt annet:</p>
       <ul>
-        <li>Mikrofoner (Håndholdt og hodebøyle)</li>
+        <li>Mikrofoner (håndholdt og hodebøyle)</li>
         <li>Prosjektor med en rekke utganger</li>
-        <li>DJ Utstyr (CDJ2000NXS og DJM2000)</li>
-        <li>Top moderne lysrigg</li>
+        <li>DJ-Utstyr (CDJ2000NXS og DJM2000)</li>
+        <li>Topmoderne lysrigg som kan brukes i både presentasjon og klubb-setting</li>
```

Comment:
*Toppmoderne (to `p`-er)
"Toppmoderne lysrigg, som kan brukes i både presentasjon- og klubbsetting"
Jeg skal ikke garantere at det ovenfor er korrekt, men jeg tror det blir mer rett mtp grammatikken og bindingen av ordene.